lib/index.js

```diff
@@ -625,6 +625,9 @@ class Generator extends EventEmitter {
         })()
           .then(function() {
             delete self.runningState;
+            const eventName = `done$${namespace || 'unknownnamespace'}#${methodName}`;
+            debug(`Emiting event ${eventName}`);
+            self.env.emit(eventName, namespace, self);
```

Comment:
Here may throw error `self.env.emit is not a function`

Reply:
Why?
self.env is an Environment instance which extends events.

Reply:
Hmm, there is something wrong with my usage, I mistakenly think that `env` is just an `Object` instance.